distributed/recreate_tasks.py

```diff
@@ -10,17 +10,19 @@
 logger = logging.getLogger(__name__)
 
 
-class ReplayExceptionScheduler:
-    """A plugin for the scheduler to recreate exceptions locally
+class ReplayTaskScheduler:
+    """A plugin for the scheduler to recreate tasks locally
 
     This adds the following routes to the scheduler
 
     *  cause_of_failure
+    *  get_runspec
     """
 
     def __init__(self, scheduler):
         self.scheduler = scheduler
         self.scheduler.handlers["cause_of_failure"] = self.cause_of_failure
+        self.scheduler.handlers["get_runspec"] = self.get_runspec
         self.scheduler.extensions["exceptions"] = self
```

Comment:
We might want to rename this extension here, e.g. `replay-tasks`?

Reply:
renamed. does this need to be changed elsewhere? kinda hard to tell where references to "exceptions" in the code base are relevant to this or not.

Reply:
I think not. If in doubt keep as is. Once you have a green CI you can still rename in the end